internal/repository/evmreader_test.go

```diff
@@ -43,6 +41,8 @@ func (s *RepositorySuite) TestInsertInputsAndUpdateLastProcessedBlock() {
 
 	response, err := s.database.GetInput(s.ctx, 5, common.HexToAddress("deadbeef"))
 	s.Require().Nil(err)
+	// necessary while the insert function does not return the generated Id
+	response.Id = 0
```

Comment:
Since you already updated the base insert functions, I think it's ok to update the get function as well to return the id

Reply:
Done in 24bbce4dd953a53a0e6a018952ce3bba50268b6d.